mp/importer/batcher.py

```diff
@@ -26,9 +25,42 @@ def run_in_batches(iterable, end_batch_callback=None, batch_size=2, batch_start=
             if position_in_batch == 0: 							# ends completed batches
                 end_batch_callback()
                 ended = True
-        if max_batches is not None and current_batch == batch_start + max_batches:					# if gets maximum allowed batches, break 
+        if max_batches is not None and current_batch == batch_start + max_batches:	# if gets maximum allowed batches, break 
             break
     if ended is False:
         end_batch_callback()	
 
 
+def parse_arguments(parser):		 						
+    """ Adds batcher arguments to the parser									 
+
+    This function gets the parser containing user inputs
+    It adds arguments and returns a parser
+    The parsing of args is performed in the main script 
+    """ 
+    parser.add_argument('--batch-size', dest='batch_size', action='store', type=int, help="Number of elements in each batch")			# set default values here? now in batcher		
+    parser.add_argument('--batch-start', dest='batch_start', action='store', type=int, help="starts importing batches from a given batch")
+    parser.add_argument('--max-batches', dest='max_batches', action='store', type=int, help="Maximum amount of batches imported")
+    return parser		 
```

Comment:
I don't think it's necessary to return the parser here. the calling function already has a reference